benchmark/SSM.hs

```diff
@@ -1,35 +1,43 @@
 module Main where
```

Comment:
I thought we had already made the changes to use a record to contain the various options?